### ISSUE TITLE
Generated @Modifiable does not compile

### ISSUE THREAD

**Issue by jsotuyod** [immutables/immutables#319]
Created a minimal example:

```java
@Value.Modifiable
public abstract class User {
	public abstract int getId();
	public abstract String getUsername();
	public abstract String getPassword();	
}
```

And the generated class under version 2.1.18 does not compile, since it refers to `ImmutableUser` for non null checks, for instance:

```java
  public ModifiableUser from(User instance) {
    ImmutableUser.requireNonNull(instance, "instance");
    setId(instance.getId());
    setUsername(instance.getUsername());
    setPassword(instance.getPassword());
    return this;
  }
```

The static method `requireNonNull` exists on the muttable class, it's just not being called.

**Comment by jsotuyod** [comment]
Just checked in the newly released 2.1.19 and the problem is gone.

**Comment by elucash** [comment]
Great! One problem less
